### PR TITLE
Explain how to deal with certificate errors

### PR DESCRIPTION
**Pull request by schultyy**
Even though this is specific to Let's Encrypt this is a valuable tutorial on how a user can manually rotate (faulty) SSL/TLS certificates on their Travis CI Enterprise instance, helping them to possibly rule
out certain sources of error.

### CODE REVIEW

File: user/enterprise/operations-manual.md [hunk 1]
```diff
@@ -126,11 +126,36 @@ Either `sudo journalctl -u travis-worker` or `sudo systemctl status travis-worke
...
 
-```sh
+```bash
 $ mkdir -p /var/tmp/travis-run.d/
 $ chown -R travis:travis /var/tmp/travis-run.d/
 ```
 
+## Travis CI Enterprise becomes unavailable do to a certificate error
+
+Travis CI Enterprise is using a Let's Encrypt certificate which has expired. A (forced) renewal via the following commands don't have any effect:
```
Comment (MariadeAnton) [inline user/enterprise/operations-manual.md]: are all Enterprise installations using certificates from Let's Encrypt? If not, we can make it generic and then use Let's Encrypt as an example. If yes, all good here! 

Could you elaborate on the forced renewal not having any effect? Is it that those are the usual commands to fix it but they may not work?

Comment (acnagy) [reply user/enterprise/operations-manual.md]: +1 for making more generic, or turning this to a blog post follow up :) I think it's likely that Let's Encrypt is a minority for enterprise customers so our docs referencing very directly. 

Or, we could split the Let's Encrypt sections in the docs out to their own doc? wdyt?

**Comment by plaindocs** [comment]
Hey @schultyy what is the status of this? Do you still need a review from me, or are you planning to generalize it according @acnagy 's suggestion?

**Comment by plaindocs** [comment]
Closing, please reopen if still relevant.

**Pull request closed without merging**